modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -411,13 +410,35 @@ public static boolean oracleExists(FluoConfiguration config) {
   }
 
   public boolean oracleExists() {
-    CuratorFramework curator = getAppCurator();
+    return oracleExists(getAppCurator());
+  }
+
+  public static int numWorkers(CuratorFramework curator) {
+    int numWorkers = 0;
     try {
-      return curator.checkExists().forPath(ZookeeperPath.ORACLE_SERVER) != null
-          && !curator.getChildren().forPath(ZookeeperPath.ORACLE_SERVER).isEmpty();
+      if (curator.checkExists().forPath(ZookeeperPath.FINDERS) != null) {
+        for (String path : curator.getChildren().forPath(ZookeeperPath.FINDERS)) {
+          if (path.startsWith("f-")) {
```

Comment:
Should make this a constant shared with other code.  This will make the implication more obvious if someone changes it.